.bazelversion

```diff
@@ -1 +1 @@
-4.1.0rc1
\ No newline at end of file
+4.1rc4
```

Comment:
Can you double check whether this should be `4.1.0rc4`?

Reply:
oops, need to fix that for the other as well